pkg/apis/machineconfiguration.openshift.io/v1/helpers.go

```diff
@@ -13,6 +13,7 @@ import (
 // It sorts all the configs in increasing order of their name.
 // It uses the Ignition config from first object as base and appends all the rest.
 // Kernel arguments are concatenated.
+// FIPS uses the last specified value.
```

Comment:
I think this will be confusing for users, especially because unset values will decode into `false`.  Like, the following would have FIPS mode OFF:
```yaml
metadata:
  name: 00_config_controller
spec:
  config: ...
---
metadata:
  name: 99_config_fips
spec:
  FIPS: true
---
metadata:
  name: 99_config_master
spec:
  osImageURL: thelatestversion
```

Reply:
We might want to go pointer then to differentiate between set/unset as well?

Reply:
Yeah, probably. Another option would be to fail closed, i.e. if ANY configs have FIPS=true, it will be turned on.